client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -137,8 +143,31 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
     public int status(@CommandLine.Option(names = {"-i", "--deploymentId"}, paramLabel = "Deployment ID",
             required = true) String deploymentId) {
 
-        LocalDeployment status = nucleusAdapterIpc.getLocalDeploymentStatus(deploymentId);
-        System.out.printf("%s: %s%n", status.getDeploymentId(), status.getStatus());
+        LocalDeploymentStatus status = nucleusAdapterIpc.getLocalDeploymentStatus(deploymentId);
+        TextStringBuilder statusBuilder = new TextStringBuilder();
+        statusBuilder.appendln(String.format(
+                "Deployment ID: %s", status.getDeploymentId()));
+        statusBuilder.appendln(String.format(
+                "Created on: %s", status.getCreatedOn()));
+        statusBuilder.appendln(String.format("Status: %s", status.getStatus()));
+        if (status.getDeploymentStatusDetails() != null) {
+            DeploymentStatusDetails deploymentStatusDetails = status.getDeploymentStatusDetails();
```

Comment:
nit: declare this variable outside your "if" so that you don't need to call getDeploymentStatusDetails twice